docs/usage_guide/integrations.rst

```diff
@@ -0,0 +1,177 @@
+Third Party Library Integrations
+================================
+
+Modin is a drop-in replacement for Pandas, so we want it to interoperate with third-party libraries just as Pandas does. 
+To see where Modin performs well and where it needs to improve, we've selected a number of important machine learning + visualization + statistics libraries, 
+and then looked at examples (from their documentation, if possible) about how they work with Pandas. Then we ran those same workflows with Modin, and 
+tracked what worked, and what failed.
+
+In the table below, you'll see, for each third-party library we tested, the number of successful test calls / total test calls, and a qualitative description of how both Pandas and Modin integrate with that library.
+
+In the deeper dive, you can view the Jupyter notebook we have used to test API calls and the corresponding Github issues filed. If you come across other issues/ examples 
+in your own workflows we encourage you to file an `issue <https://github.com/modin-project/modin/issues/new/choose>`_ or contribute a `PR <https://github.com/modin-project/modin/pulls>`_!
+
+
+.. note::
+    These interoperability metrics are preliminary and not all APIs for each library have been tested. Feel free to add more!
+
+
+Modin Interoperability by Library
+'''''''''''''''''''''''''''''''''
+.. list-table::
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - Library
+     - API successes / calls
+     - Interoperability
+     
+   * - seaborn
+     - 73% (11/15)
+     - **Pandas**: Accepts Pandas DataFrames as inputs for producing plot |br|
+       **Modin**: Mostly accepts Modin DataFrames as inputs for producing plots, but fails in some cases (pairplot, lmplot), and for others (catplot, objects.Plot) works for some parameter combinations  but fails with others
```

Comment:
I think this sentence is a run-on. The part

> Mostly accepts Modin DataFrames as inputs for producing plots, but fails in some cases (pairplot, lmplot),

makes sense, but I get confused when I see the next part

> and for others (catplot, objects.Plot) works for some parameter combinations  but fails with others

Is it possible to rewrite this more clearly? I am guessing we want to say something like 

```suggestion
       **Modin**: Mostly accepts Modin DataFrames as inputs for producing plots, but fails completely in some cases (pairplot, lmplot), and for others (catplot, objects.Plot) works only for some parameter combinations
```

Reply:
Yes that sentence reads a bit unclear - I fixed it in the recent commit!